### PR TITLE
chore: use `@php` in Blade templates

### PR DESCRIPTION
**Pull request by davwheat**
**Changes proposed in this pull request:**

As per title.

**Reviewers should focus on:**

None

**Necessity**

- [x] Has the problem that is being solved here been clearly explained?
- [x] If applicable, have various options for solving this problem been considered?
- [x] For core PRs, does this need to be in core, or could it be in an extension?
- [x] Are we willing to maintain this for years / potentially forever?

**Confirmed**

- [ ] Frontend changes: tested on a local Flarum installation.
- [x] Backend changes: tests are green (run `composer test`).
- [x] Core developer confirmed locally this works as intended.
- [x] Tests have been added, or are not appropriate here.

### CODE REVIEW

**Comment by askvortsov1** [comment]
Eh, not sure I agree with this. If we're taking action here, we might as well fix the issue.

**Comment by davwheat** [comment]
I think "issue" is stretching it. As far as I can see, `@php` is just alternate syntax to `<?php`, isn't it?

File: views/layouts/basic.blade.php [hunk 1]
```diff
@@ -1,9 +1,8 @@
-{{-- TODO: Change below to @php when Laravel is upgraded --}}
 <?php
   $primaryColor = $settings->get('theme_primary_color', '#000');
 ?>
```
Comment (davwheat) [inline views/layouts/basic.blade.php:3]:
```suggestion
@php
  $primaryColor = $settings->get('theme_primary_color', '#000');
@endphp
```

**Review by askvortsov1 — APPROVED** [review]

File: views/layouts/basic.blade.php [hunk 2]
```diff
@@ -1,9 +1,8 @@
-{{-- TODO: Change below to @php when Laravel is upgraded --}}
-<?php
+@php
   $primaryColor = $settings->get('theme_primary_color', '#000');
-?>
+@endphp
```
Comment (SychO9) [inline views/layouts/basic.blade.php:3]: from local testing it seems like declaring variables within `@php` doesn't actually work for some reason 🤔

Comment (SychO9) [reply views/layouts/basic.blade.php:3]: oh, it works without the comment on top nvm

**Review by SychO9 — APPROVED** [review]

**Pull request merged**